src/components/PageSidebar/SidebarTabSharing.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+	<div class="shares-container">
+		<!-- loading -->
+		<NcEmptyContent v-if="loading('shares')">
+			<template #icon>
+				<NcLoadingIcon />
+			</template>
+		</NcEmptyContent>
+
+		<!-- shares list -->
+		<ul v-else class="sharing-list">
+			<SharingEntryLink v-if="!shares.length" />
+			<SharingEntryLink v-for="(share, index) in shares"
+				v-else
```

Comment:
I think you can drop this. if shares is empty the `v-for` will not do anything anyway.

Reply:
I see. Still, isn't it better to make explicit in the code that either the first, or the second is active?

Reply:
:shrug: I don't know and trust your choice.